R/shiny_fuse.R

```diff
@@ -0,0 +1,651 @@
+#' shiny_fuse
+#'
+#' TODO
+#'
+#' @param out_annofuse TODO
```

Comment:
I think the input file needs to have columns : "LeftBreakpoint", "RightBreakpoint", "FusionName", "Gene1A" and "Gene1B" for the domain plots to work. Should we check for these columns and error out saying please provide required columns?

Reply:
That's an excellent point.
It actually calls for a function that checks that *all* required columns (and names) are in.
Could be an internal function that returns nothing if all is fine (well, at most a message) - and an error/warning, depending on the context/how "bad" that would be.

Reply:
sounds good, I can add a small function to check that.